front/app/containers/Admin/sideBar/index.tsx

```diff
@@ -73,7 +73,7 @@ const IconWrapper = styled.div`
   justify-content: center;
 `;
 
-const Text = styled.div`
+const CustomText = styled.div`
```

Comment:
Renaming the usage of custom text here. I'm not changing the usages in here to the cl2 library `Text` component as it is out of scope but hopefully we can come back to it soon